source/platform/renderers/SFML2/SFML2.cpp

```diff
@@ -16,119 +16,155 @@
 #ifdef __APPLE__
 # include <OpenGL/gl.h>
 #else
+#ifdef _WIN32
+#include <Windows.h>
+#endif
 # include <GL/gl.h>
 #endif
 
 #include <cmath>
-#include <unistd.h>
```

Comment:
Why has this been removed?

Reply:
This file could not be found, in visual studio. I meant to add ifndef _WIN32 around it as it appears to be for POSIX systems.

Reply:
Interesting. I didn't know this had broken the Windows build of SFML2. I work on a Mac.